web/js/jquery.gerbera.editor.js

```diff
@@ -162,7 +162,7 @@
     if (item) {
       reset(modal);
       if (item.image) {
-        modal.find('#mediaimage').prop('src', item.image.value);
+        modal.find('#mediaimage').prop('src', '//' + item.image.value);
```

Comment:
What's the background?

`image` is always the full path starting with virtual Url?

Reply:
Yes, but Firefox refused to load it without a protocol element (http://) so I used the "//" which means match the pages protocol

Reply:
What's your virtualUrl setting in config?

Reply:
Good question, I'm on holiday for a few days

Reply:
Enjoy it!

Reply:
Did you check you config. The change breaks my WebUI

Reply:
So, I have virtualURL as `hostname:port` in config.

I guess you have `http://hostname:port` which would have broken.

I will put together a PR to fix it.